# HG changeset patch
# Parent 2a1ce2b7af72812b78dcfba3c7501905b1a8be95
bug 41216 - editor context menu failure
part 1, fixes getting the context menu event on a plugin

diff --git a/content/events/src/nsEventStateManager.cpp b/content/events/src/nsEventStateManager.cpp
--- a/content/events/src/nsEventStateManager.cpp
+++ b/content/events/src/nsEventStateManager.cpp
@@ -1840,11 +1840,13 @@ nsEventStateManager::FireContextClick()
                              type == NS_FORM_INPUT_FILE ||
                              type == NS_FORM_TEXTAREA);
       }
+#if 0 // ACTIVESTATE we want to allow scintilla context menus
       else if (tag == nsGkAtoms::applet ||
                tag == nsGkAtoms::embed  ||
                tag == nsGkAtoms::object) {
         allowedToDispatch = PR_FALSE;
       }
+#endif
     }
 
     if (allowedToDispatch) {
